test/intl402/DateTimeFormat/taint-Object-prototype-quarter.js

```diff
@@ -0,0 +1,16 @@
+// Copyright 2019 Google Inc.All rights reserved.
```

Comment:
small typo repeated in many test files: 

s/Inc.All/Inc. All/

(space before All)

Reply:
DONE